src/devices/Button/ButtonBase.cs

```diff
@@ -78,17 +80,24 @@ public ButtonBase()
         /// </summary>
         /// <param name="doublePress">Max ticks between button presses to count as doublepress.</param>
         /// <param name="holding">Min ms a button is pressed to count as holding.</param>
-        public ButtonBase(TimeSpan doublePress, TimeSpan holding)
+        /// <param name="debounceTime">The amount of time during which the transitions are ignored, or zero</param>
+        public ButtonBase(TimeSpan doublePress, TimeSpan holding, TimeSpan debounceTime = default(TimeSpan))
```

Comment:
if `debounceTime * 2 >= doublePress` then you will never get doublePress event, you should throw if that's the case. I'd recommend to even make it `3*` since timing clicks perfectly would be literally impossible